src/notificationmanager.cpp

```diff
@@ -771,3 +768,69 @@ void NotificationManager::slotNgfEventFinished(quint32 id)
         m_ngfEvent = 0;
 }
 
+void NotificationManager::slotVoicemailWaitingChanged(bool waiting)
+{
+    const QString category(QStringLiteral("x-nemo.messaging.voicemail-waiting"));
```

Comment:
Do we have a named constant for `"x-nemo.messaging.voicemail-waiting"` in some headers? If not, should we have it and use it here?